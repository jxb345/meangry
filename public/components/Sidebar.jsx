import React from "react";

const Sidebar = () => {
  return (
    <div>
      <div className="rules-title">IMPORTANT</div>
      <div className="rules">
        Do <strong>NOT</strong> use or include: <br />
        &#8226; personal information <br />
        &#8226; names, locations, etc. <br />
        &#8226; bigoted language of any kind <br />
        &#8226; threatening, violent language <br />
      </div>
    </div>
  );
};

export default Sidebar;
