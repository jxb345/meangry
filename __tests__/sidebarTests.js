import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Sidebar from '../public/components/Sidebar.jsx';
