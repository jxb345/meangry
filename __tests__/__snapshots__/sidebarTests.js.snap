// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`preview-snapshot-test 1`] = `
<div>
  <div
    className="rules-title"
  >
    IMPORTANT
  </div>
  <div
    className="rules"
  >
    Do 
    <strong>
      NOT
    </strong>
     use or include: 
    <br />
    • personal information 
    <br />
    • names, locations, etc. 
    <br />
    • bigoted language of any kind 
    <br />
    • threatening, violent language 
    <br />
  </div>
</div>
`;
